content/manuals/desktop/networking.md

```diff
@@ -184,3 +184,29 @@ to give the container direct access to the network stack of the host.
 
 See the [run command](/reference/cli/docker/container/run.md) for more details on
 publish options used with `docker run`.
+
+### IPv4-only, Dual-stack and IPv6-only (beta)
+
+> [!NOTE]
+> 
+> This feature is under active development and the exact semantics of all three
+> options might change in future releases.
+
+Starting with v4.35, Docker Desktop has new options to configure how the VM behaves and the default
+settings applied to Docker networks.
+
+With IPv4-only, the DNS resolver embedded in the VM will only serve A (= IPv4) records and suppress
+any AAAA (= IPv6) records. This might be useful if some of your containerized software try to
+connect to IPv6 addresses but your host doesn't have proper IPv6 connectivity.
+
+With Dual-stack, both A and AAAA records are served by the DNS resolver. Docker networks are
```

Comment:
```suggestion
With Dual-stack IPv4 and IPv6, both A and AAAA records are served by the DNS resolver. Docker networks are
```

Reply:
I'd prefer to stick with the option name shown in the UI.